client/public/materializeCSS/materialize_style.css

```diff
@@ -6,6 +6,22 @@
  * Made By MaterializeCSS.com
  */
 
+ input {
+  font-family: 'Amatic SC', cursive";
+ }
+
+ .loginbox, .signupform {
+  background: white;
+  opacity: .8;
+  padding: 20px;
+  margin-top: 10% !important;
```

Comment:
`!important` is usually a red flag. Not a big issue here but keep in mind that there are usually alternative ways of providing greater specificity.

Reply:
```suggestion
  margin-top: 10%;
```